crates/sui-indexer-alt-graphql/src/api/types/transaction_kind/change_epoch.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql::Object;
+use sui_types::transaction::ChangeEpoch as NativeChangeEpoch;
+
+#[derive(Clone)]
+pub(crate) struct ChangeEpochTransaction {
+    pub(crate) native: NativeChangeEpoch,
+}
+
+// TODO(DVX-1372): Support systemPackages when MovePackage is available.
```

Comment:
`MovePackage` should already be available, unless I'm missing something (it's required by the replay tool).